README.md

```diff
@@ -42,54 +42,46 @@ This project uses the following Style Guidelines
     .
     ├── ...
     ├── src                     # Source files (This is where the API resides)
-    │   ├── config/             # Contains config file for MySQL DB and PM2 configuration
     │   ├── controller/         # All API logic resides here.
     │   ├── model/              # All Mongoose Schemas, models, and database logic.
     │   ├── middleware/         # Authentication middleware resides inside this folder. 
-    │   ├── routes.js           # This is where routes get created and get tied back to their specific controller. 
+    │   ├── routes.config.js    # This is where routes get created and get tied back to their specific controller. 
     │   └── index.js            # Where everything begins.
     └── ...
 
 ## Setup
 
 ### Step 1. Install required node modules
 
-```sh
-    cd cve-services
-    npm install
-```
+This assumes the latest `node` and `npm` are installed.
+
+    ```sh
+        cd cve-services
+        npm install
+    ```
 
 ### Step 2. Setup and start MongoDB locally
 
     Install MongoDB locally
-- https://docs.mongodb.com/manual/administration/install-community/
+    - https://docs.mongodb.com/manual/administration/install-community/
 
     Download MongoDB Compass (MongoDB GUI)
-- https://www.mongodb.com/download-center/compass
-
-### Step 3. Edit the src/config/config.json file to suite your requirements
-
-    This config file is used to config variables for MongoDB
+    - https://www.mongodb.com/download-center/compass
 
-### Step 4. Create a .env file in the root folder
-
-    This env file contains important variables needed.
-    See .env-example for an example
-
-### Step 5. Start the node application
+### Step 3. Start the node application
 
 In order to start a dev enviorment:
 
-```sh
-    npm run start:dev
-```
-
-In order to start a test enviorment with unit testing:
+    ```sh
+        npm run start:dev
+    ```
 
-```sh
-    npm run start:test
-```
+In order to start a test environment with unit testing:
 
+    ```sh
+        npm run start:test
+    ```

```

Comment:
Remove this since it's also mentioned in the Unit Testing section in the README.